source/blog/025-request-for-comments-nested-map-functions.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Request for Comments: Nested Map Functions
+author: Natalie Weizenbaum
+date: 2020-9-16 14:40 PST
+---
+
+As Sass libraries get more complex and have more users with different needs,
+they tend to develop the need for configuration. This configuration is often
+hierarchical, and ends up being represented as maps that contain maps that
+contain still more maps. Up until now, Sass's map functions haven't really made
+it easy to work with this sort of nested map structure. But that's changing with
+the latest language proposal, written by Sass core team member [Miriam
+Suzanne].
```

Comment:
It could be worth slipping in an eye-catching phrase to further motivate this (something about design tokens/systems?)

Reply:
Good call, done.